README.md

```diff
@@ -150,6 +168,19 @@ Note [the process requires multiple steps](https://github.com/NHSDigital/dtos-de
 - Consider switching on format-on-save in your editor (e.g. [Black](https://github.com/psf/black) for python)
 - (Internal contributions only) contact the `#screening-manage` team on slack with any questions
 
+
+## import into terraform state file
```

Comment:
Can we move to https://github.com/NHSDigital/dtos-manage-breast-screening/tree/main/docs ?
Maybe infra-faq.md ?